<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_kala</class>
 <widget class="QDialog" name="add_kala">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1241</width>
    <height>688</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 61);</string>
  </property>
  <widget class="QCheckBox" name="checkBox_color">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>254</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 170, 0);
background-color: rgb();</string>
   </property>
   <property name="text">
    <string>Color:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_color">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>270</y>
     <width>311</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(209, 209, 209);
background-image: url();</string>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Red</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Blue</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Yellow</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Other</string>
    </property>
   </item>
  </widget>
  <widget class="QCheckBox" name="checkBox_weight">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>345</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 170, 0);
background-color: rgb();</string>
   </property>
   <property name="text">
    <string>Weight:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_gennder">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>545</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 170, 0);
background-color: rgb();</string>
   </property>
   <property name="text">
    <string>Gender:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_brand">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>435</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 170, 0);
background-color: rgb();</string>
   </property>
   <property name="text">
    <string>Brand:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxgender">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>565</y>
     <width>311</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(209, 209, 209);
background-image: url();</string>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Men</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Women</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Other</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_brand">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>455</y>
     <width>311</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(209, 209, 209);
background-image: url();</string>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Nike</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Addias</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Puma</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Other</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEdit_weight">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>360</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>440</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 170, 0);
background-color: rgb();</string>
   </property>
   <property name="text">
    <string>Name :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_name">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>450</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>550</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 170, 0);
background-color: rgb();</string>
   </property>
   <property name="text">
    <string>Price:</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_Equipment">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>350</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb();
color: rgb(255, 170, 255);</string>
   </property>
   <property name="text">
    <string>Equipment</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_cloths">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>350</y>
     <width>191</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb();
color: rgb(255, 170, 255);</string>
   </property>
   <property name="text">
    <string>Clothes and shoes</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 170, 0);
background-color: rgb();</string>
   </property>
   <property name="text">
    <string>Product Type:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_price">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>560</y>
     <width>311</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="maximum">
    <number>999999999</number>
   </property>
  </widget>
  <widget class="QWidget" name="widget_photo" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>201</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_addphoto">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>230</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 61);
background-image: url(:/new/prefix1/add photo.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_back">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 61);
background-image: url(:/new/prefix1/Back.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>170</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 170, 0);
background-color: rgb();</string>
   </property>
   <property name="text">
    <string>Number :</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_number">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>180</y>
     <width>311</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(209, 209, 209);
background-image: url();</string>
   </property>
   <property name="maximum">
    <number>999999999</number>
   </property>
  </widget>
  <widget class="QPushButton" name="push_buttonconfirm">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>640</y>
     <width>179</width>
     <height>33</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/confirmation.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxgender_mat">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>565</y>
     <width>311</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(209, 209, 209);
background-image: url();</string>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>woody</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>glassy</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ferrous</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
